Dockerfile

```diff
@@ -6,7 +6,8 @@ MAINTAINER KBase Developer
 # install line here, a git checkout to download code, or run any other
 # installation scripts.
 
-# RUN apt-get update
```

Comment:
Needs tests as you mentioned earlier, also needs release notes.

Reply:
Added tests and release notes